src/applications/vaos/tests/new-appointment/components/VAFacilityPage/VAFacilityPageV2.multi.unit.spec.jsx

```diff
@@ -180,6 +185,53 @@ describe('VAOS integration: VA flat facility page - multiple facilities', () =>
     );
   });
 
+  it('should show residential address and sort by distance if we have coordinates', async () => {
+    mockParentSites(parentSiteIds, [parentSite983, parentSite984]);
+    mockDirectBookingEligibilityCriteria(parentSiteIds, directFacilities);
+    mockRequestEligibilityCriteria(parentSiteIds, requestFacilities);
+    mockFacilitiesFetch(vhaIds.join(','), facilities);
+    mockEligibilityFetches({
+      siteId: '983',
+      facilityId: '983',
+      typeOfCareId: '323',
+    });
+    const store = createTestStore({
+      ...initialState,
+      user: {
+        ...initialState.user,
+        profile: {
+          ...initialState.user.profile,
+          vet360: {
+            residentialAddress: {
+              addressLine1: 'PSC 808 Box 37',
+              city: 'FPO',
+              stateCode: 'AE',
+              zipCode: '09618',
+              latitude: 37.5615,
+              longitude: -121.9988,
+            },
+          },
+        },
+      },
+    });
+    await setTypeOfCare(store, /primary care/i);
+
+    const screen = renderWithStoreAndRouter(<VAFacilityPage />, {
+      store,
+    });
+
+    await screen.findByText(
+      /Choose a VA location for your Primary care appointment/i,
+    );
+    expect(screen.baseElement).to.contain.text(
+      'We base this on the address we have on file',
+    );
+    expect(screen.baseElement).to.contain.text('Your address on file');
+    expect(screen.baseElement).to.contain.text('PSC 808 Box 37');
+    expect(screen.baseElement).to.contain.text('FPO, AE 09618');
+    expect(screen.baseElement).to.contain.text(' miles');
```

Comment:
Should we also check that facilities are appropriately sorted? I set that up on my EC sorting ticket, should be somewhat similar.

Reply:
Added test to check that the first facility is the closest